### ISSUE TITLE
Nice :)

### ISSUE THREAD

**Issue by pchaganti** [fungusakafungus/cloudformation-jsonschema#1]
This is very interesting. Have manually tried to put together a schema for CF. This is much better. Are you currently using this in any project?

thanks!

**Comment by fungusakafungus** [comment]
No, I do not use this. It is a proof of concept type of thing. When the schema is complete, I plan to generate python classes for a dsl a la troposphere.
